src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,17 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
```

Comment:
You don't need to change it because you need this controller for future development, but the controller name must be related to the code, that is, GreetingController.
👍